fdbserver/workloads/BulkLoading.actor.cpp

```diff
@@ -184,18 +195,17 @@ struct BulkLoading : TestWorkload {
 				wait(tr.onError(e));
 			}
 		}
-		return true;
+		return std::make_pair(errorTasks, true);
 	}
 
-	ACTOR Future<Void> waitUntilAllTaskComplete(BulkLoading* self, Database cx) {
+	ACTOR Future<std::vector<BulkLoadTaskState>> waitUntilAllTaskCompleteOrError(BulkLoading* self, Database cx) {
 		loop {
-			bool complete = wait(self->checkAllTaskComplete(cx));
-			if (complete) {
-				break;
+			std::pair<std::vector<BulkLoadTaskState>, bool> res = wait(self->checkAllTaskCompleteOrError(cx));
+			if (res.second) { // If all tasks are complete or error
```

Comment:
Its a bit odd checking second argument rather than first for error?

Reply:
Agree. I will include the change in the next PR.

Reply:
Ok